### PR TITLE
Add timeout on build + begin to fix install on windows

### LINKED ISSUE #186: Setup a timeout for the Jenkins build
30 mins should be ok

### LINKED ISSUE #187: Use the right AdvancedInstaller exe
To have the logging on error. The current one only display popups

### PR DESCRIPTION
**Pull request by VincentBlondeau**
Close #187 
Close #186

### CODE REVIEW

File: windows/build-launcher-installer.bat [hunk 1]
```diff
@@ -20,5 +20,5 @@ MOVE Pharo windows\Pharo-win\Pharo
 
 cd windows
 REM advinst.exe /newproject $ADVINST_PROJECT -lang en -overwrite
```
Comment (guillep) [inline windows/build-launcher-installer.bat]: Should this REM line be changed also? It looks like a log of the following line.

File: Jenkinsfile [hunk 2]
```diff
@@ -5,10 +5,12 @@ properties([disableConcurrentBuilds()])
...
-  buildArchitecture('32', '61')
-  buildArchitecture('64', '61')
-  buildArchitecture('32', '70')
-  buildArchitecture('64', '70')
+  timeout(time: 30, unit: 'MINUTES') { 
+    buildArchitecture('32', '61')
+    buildArchitecture('64', '61')
+    buildArchitecture('32', '70')
+    buildArchitecture('64', '70')
+  }
```
Comment (guillep) [inline Jenkinsfile]: ok!

**Comment by guillep** [comment]
@VincentBlondeau Can you explain a bit better the change? Are there two executables one that is command line and one that is a GUI app? And we are changing to the one that uses the command line and does not hang?

If it is so, this is SUPER GOOD!

File: windows/build-launcher-installer.bat (hunk 1 above)
Comment (VincentBlondeau) [reply windows/build-launcher-installer.bat]: Yup

**Comment by VincentBlondeau** [comment]
I don't know exactly how this software is conceived, but AFAIK, the advinst.exe is a GUI mode, so display a popup on errors, and AdvancedInstaller.com is a cmd line exec. So, it will not hang and display a "magnificent" error message on failure ;)

**Pull request merged**